modules/node-tuning-operator-supported-tuned-daemon-plug-ins.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/using-node-tuning-operator.adoc
+
+[id='supported-tuned-daemon-plug-ins-{context}']
+= Supported Tuned daemon plug-ins
+
+Excluding the `[main]` section, the following Tuned plug-ins are supported when
+using custom profiles defined in the `profile:` section of the Tuned CR:
+
+* audio
+* cpu
+* disk
+* eeepc_she
+* modules
+* mounts
+* net
+* scheduler
+* scsi_host
+* selinux
+* sysctl
+* sysfs
+* usb
+* video
+* vm
+
+There is some dynamic tuning functionality provided by some of these plug-ins
+that is not supported.
+
```

Comment:
Can these two sentences, line 27 and 30, be combined?

Reply:
Don't see why not. Let's combine them.